joy/joy_node.cpp

```diff
@@ -50,8 +50,8 @@ void on_parameter_event(const rcl_interfaces::msg::ParameterEvent::SharedPtr eve
       g_scale_angular = changed_parameter.value.double_value;
       std::cout << "Changed scale_angular to " << g_scale_angular;
     } else {
-      printf("Warning: ignoring attempt to set parameter %s with type %d\n",
-        changed_parameter.name.c_str(), changed_parameter.value.type);
+      // printf("Warning: ignoring attempt to set parameter %s with type %d\n",
+      //   changed_parameter.name.c_str(), changed_parameter.value.type);
```

Comment:
Ok. I'd have to look at the Parameter API's current state, but I think there should be a better way to update the parameters other than listening to the ParameterEvents, which also doesn't have this problem. This is probably fine for now.